h/activity/views.py

```diff
@@ -40,6 +41,7 @@ def search(request):
         'total': result.total,
         'aggregations': result.aggregations,
         'timeframes': result.timeframes,
+        'page': paginate_es(request, result),
```

Comment:
I'm not sure that this is how this will actually work in the end (passing the already-fetched search result into `paginate_es()`) because if it's the paginator that figures out what the offset and limit will be, then we need to call the paginator _before_ actually executing the search query, because we need to apply the offset and limit to the query before executing it. But for now this gives the paginator what it needs to generate the template data needed to render the pagination control (it needs to know the URL params and the total number of search results).